patches/tsdx+0.13.2.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/node_modules/tsdx/dist/babelPluginTsdx.js b/node_modules/tsdx/dist/babelPluginTsdx.js
+index 57d9c93..418a103 100644
+--- a/node_modules/tsdx/dist/babelPluginTsdx.js
++++ b/node_modules/tsdx/dist/babelPluginTsdx.js
+@@ -73,10 +73,6 @@ exports.babelPluginTsdx = rollup_plugin_babel_1.default.custom(() => ({
+             { name: '@babel/plugin-proposal-optional-chaining' },
+             // Adds syntax support for default value using ?? operator
+             { name: '@babel/plugin-proposal-nullish-coalescing-operator' },
+-            {
+-                name: '@babel/plugin-transform-regenerator',
+-                async: false,
+-            },
```

Comment:
suprisingly clean!

Reply:
@cowboyd I suspect it's because jest is included with tsdx. It has the same types as mocha and it's probably loaded alphabetically which causes mocha types to complain.